terminal_client.go

```diff
@@ -129,15 +141,30 @@ func (c *TerminalClient) listCmd() error {
 	}
 	tbl.PrintHeader()
 
-	for _, deploy := range deploys {
-		tbl.PrintRow(
-			deploy.Id,
-			deploy.Pid,
-			yn(deploy.Tracked),
-			deploy.Port,
-			deploy.Health,
-			fmt.Sprintf("%v", deploy.Errors),
-		)
+	var deployIds []string
+	for k := range deployTable {
+		deployIds = append(deployIds, k)
+	}
+	sort.String(deployIds)
+
+	for _, deployId := range deployIds {
```

Comment:
nit: I think it's simpler to just sort the table, and then when printing it to omit any duplicate deploy ids.